recipe/meta.yaml

```
@@ -12,14 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1000
+  script: python -m pip install --no-deps --ignore-installed .
```

```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```

this is what is actually suggested in the new example in staged-recipies https://github.com/conda-forge/staged-recipes/blob/3077038383d645912b3b4a5ade1a8ce4df55ea76/recipes/example/meta.yaml#L33

I think it's because of a behavior change in `pip` WRT dependencies, but I didn't have time to investigate further.

thanks for the PR @xhochy!